src/Faithlife.PortableContrib/Text/NormalizationForm.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace System.Text
+{
+	public enum NormalizationForm

```

Comment:
You need to add this type to [TypesForwarded.cs](https://github.com/scott-fleischman/PortableContrib/blob/master/src/Faithlife.PortableContrib/TypesForwarded.cs).